tests/unit_tests/data_steward/tools/run_deid_test.py

```diff
@@ -1,9 +1,26 @@
+"""
+Unit test for run_deid module
+
+test_parse_args -- ensures the output dataset name includes _deid
+test_main -- ensures the parameter list contains the output dataset command line argument
+test_known_tables -- ensures all table names known to curation are returned
+test_get_output_table_schemas -- ensures only table schemas for suppressed tables are copied
+
+Original Issue: DC-744
+"""
+
+# Python imports
 import unittest
+import os
+from argparse import Namespace, ArgumentTypeError
 
 # Third party imports
 from mock import mock, patch
 
+# Project imports
 from tools import run_deid
+from resources import DEID_PATH
+from tools.run_deid import parse_args, main
```

Comment:
redundant import.  you can access parse_args and main through the `run_deid` import above.  you would replace calls to `parse_args` with `run_deid.parse_args` and `main` with `run_deid.main`.